modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/metal/js/components/RuleEditor/RuleEditor.es.js

```diff
@@ -419,8 +443,17 @@ class RuleEditor extends Component {
 			}
 		);
 
+		actions.forEach(
+			(action, index) => {
```

Comment:
Maybe we could change for: `({target}, index) => this._updateCalculatorFields(index, target);`